addons/binding/org.openhab.binding.zoneminder/src/main/java/org/openhab/binding/zoneminder/handler/ZoneMinderServerBridgeHandler.java

```diff
@@ -1051,8 +1051,8 @@ public void onBridgeConnected(ZoneMinderServerBridgeHandler bridge, IZoneMinderC
                 taskPriorityRefreshData = null;
             }
 
-            // Only start if it is not running and Priority Frequency is higher than ordinary
-            if ((taskPriorityRefreshData == null) && (refreshFrequency > 1)) {
+            // Only start if Priority Frequency is higher than ordinary
+            if (refreshFrequency > 1) {
```

Comment:
The taskPriorityRefreshData is either null or set to null on line 1051, so the comment also did not seem to make much sense so I altered it.